modules/restore-replace-stopped-etcd-member.adoc

```diff
@@ -279,3 +279,11 @@ etcd-ip-10-0-133-53.ec2.internal                 3/3     Running     0
 etcd-ip-10-0-164-97.ec2.internal                 3/3     Running     0          123m
 etcd-ip-10-0-154-204.ec2.internal                3/3     Running     0          124m
 ----
++
+If the new etcd pod does not roll out automatically you can manually force an etcd redeployment. In a terminal that has access to the cluster as a cluster-admin user, run the following command:
+[source,terminal]
+----
+$ oc patch etcd cluster -p='{"spec": {"forceRedeploymentReason": "recovery-'"$( date --rfc-3339  =ns )"'"}}' --type=merge <1>
```

Comment:
I don't think that there should be any spaces between `--rfc-3339` and `=ns` in this command.